### PR TITLE
fixes js errors and tests settings

### LINKED ISSUE #28: add WYSIWYG editor to settings

### PR DESCRIPTION
**Pull request by balesh2**
Fixes #28

### CODE REVIEW

**Comment by straleyb** [comment]
:rocket:

File: app/assets/javascripts/blacklight-maps-browse.js [hunk 1]
```diff
@@ -71,14 +71,8 @@
       var success = function (position) {
         locationMarker.setLatLng([position.coords.latitude, position.coords.longitude]);
 
-        if(markers.getBounds().contains(locationMarker.getLatLng())) {
-          onCampus = true;
-        }
-        if(onCampus && !initialized){
+        if(!initialized){
```
Comment (terrellt) [inline app/assets/javascripts/blacklight-maps-browse.js]: Does this do anything anymore?

Comment (balesh2) [inline app/assets/javascripts/blacklight-maps-browse.js]: I don't think so.

Comment (terrellt) [inline app/assets/javascripts/blacklight-maps-browse.js]: :skull: it

**Pull request merged**